app/views/shared/_posts.html.erb

```diff
@@ -12,9 +12,11 @@
               </div>            
             <% end %>
           </h4>
-          <div class="to-member-card__header__text__activity">
-            <%= link_to post.member, post.user  %>
-          </div>
+          <% if current_user && current_organization %>
```

Comment:
I think it would be better like 

` <% if current_user && current_organization == post.organization %>`

With your code a logged member of any Organitzation can see names of others organizations. Can you test it?